workflow-dev/tpl/storage.sh

```diff
@@ -8,10 +8,10 @@ else
 fi
 
 database_location=`grep "database_location\s*=\s*" $HELM_GENERATE_DIR/tpl/generate_params.toml | cut -d "=" -f2 | tr -dc '[:alnum:]-\n\r'`
-if [ "$DATABASE_LOCATION" = "off-cluster" ] || [ "$database_location" = "off-cluster" ]; then
-  rm -rf $HELM_GENERATE_DIR/manifests/deis-database-*
-else
+if [ "$DATABASE_LOCATION" = "on-cluster" ] || ( [ -z "$DATABASE_LOCATION" ] && [ "$database_location" = "on-cluster" ] ); then
   helmc tpl -d $HELM_GENERATE_DIR/tpl/generate_params.toml -o $HELM_GENERATE_DIR/manifests/deis-database-rc.yaml $HELM_GENERATE_DIR/tpl/deis-database-rc.yaml
   cp $HELM_GENERATE_DIR/tpl/deis-database-service* $HELM_GENERATE_DIR/manifests/
+else
+  rm -rf $HELM_GENERATE_DIR/manifests/deis-database-*
```

Comment:
For all changes in this file, refer to comment https://github.com/deis/charts/pull/301/files#r69784495